bee-ledger/src/workers/snapshot/download.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::workers::snapshot::error::Error;
+
+use log::{info, warn};
+
+use std::{fs::File, io::copy, path::Path};
+
+pub(crate) async fn download_snapshot_file(file_path: &Path, download_urls: &[String]) -> Result<(), Error> {
```

Comment:
Can we change the function signature to something like that:

`pub(crate) async fn download_snapshot_file(file_path: &Path, download_url: String) -> Result<(), Error>`

The  `download_snapshot_file(...)` will be called 2 times, one for delta and one for full snapshot.

Where `download_url` already links to the exact file to download. The full url to the file.
And `file_path` specifies the location where to save the snapshot locally.

By this, `download_url` and `file_path` are decoupled. A user then can use a different name for the snapshot in the config.toml.

Reply:
Okey yes will create a issue for that^